cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -1819,6 +1844,19 @@ pOutputFile =
       <> Opt.completer (Opt.bashCompleter "file")
       )
 
+pNodeSocketPath :: Parser (Maybe SocketPath)
+pNodeSocketPath =
+  optional $ fmap SocketPath $
+    Opt.strOption $ mconcat
+      [ Opt.long "node-socket-path"
```

Comment:
Let's maintain consistency and leave this as `"socket-path"` as `cardano-node`, `cardano-chairman` and `cardano-submit-api`  parse for `"socket-path"`

Reply:
Done!